lib/graph/projectGraphBuilder.js

```diff
@@ -58,36 +64,37 @@ function validateNode(node) {
 /**
  * Node Provider interface
  *
- * @interface NodeProvider
+ * @interface @ui5/project/graph/ProjectGraphBuilder~NodeProvider
  */
 
 /**
  * Retrieve information on the root module
  *
  * @function
- * @name NodeProvider#getRootNode
+ * @name @ui5/project/graph/ProjectGraphBuilder~NodeProvider#getRootNode
  * @returns {Node} The root node of the dependency graph
  */
 
 /**
  * Retrieve information on given a nodes dependencies
  *
  * @function
- * @name NodeProvider#getDependencies
+ * @name @ui5/project/graph/ProjectGraphBuilder~NodeProvider#getDependencies
  * @param {Node} The root node of the dependency graph
  * @returns {Node[]} Array of nodes which are direct dependencies of the given node
  */
 
 /**
- * Generic helper module to create a [@ui5/project.graph.ProjectGraph]{@link module:@ui5/project.graph.ProjectGraph}.
+ * Generic helper module to create a [@ui5/project/graph/ProjectGraph]{@link @ui5/project/graph/ProjectGraph}.
  * For example from a dependency tree as returned by the legacy "translators".
  *
  * @public
- * @alias module:@ui5/project.graph.projectGraphBuilder
- * @param {NodeProvider} nodeProvider
- * @returns {module:@ui5/project.graph.ProjectGraph} A new project graph instance
+ * @function default
+ * @static
+ * @param { NodeProvider } nodeProvider
```

Comment:
remove spaces after `{` and before `}`

Reply:
replace `NodeProvider` with `@ui5/project/graph/ProjectGraphBuilder~NodeProvider` and make the interface public